src/eulerevaluator.cpp

```diff
@@ -41,25 +41,26 @@ void EulerEvaluator::InitializeEquationParts() {
 NetworkState EulerEvaluator::GetNextNetworkState() {
 	iterations++;
 	auto oldState = mState.DeepCopy();
+	double c1 = 0;
+	double c2 = 0;
 	for (auto &specie : equationParts) {
 		double diff = 0;
```

Comment:
diff bliver kun summeret inde i scopet der starter på linje 46. Værdien af c i Kahan's algoritme burde blive reset hver gang du starter på en ny summation. 

Du burde i stedet for declare c1 lige før eller efter du declarer diff.

Reply:
men c2 kan vel blive stående, hvor den er?

Reply:
c2 skal faktisk også med ind. For hver iteration af for-løkken har du jo med en anden species at gøre, så den fejl du bruger burde også være anderledes.